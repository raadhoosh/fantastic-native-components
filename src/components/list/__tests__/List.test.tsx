import List from "../index";
import { shallow, ShallowWrapper } from "enzyme";
import React from "react";
// tslint:disable-next-line:no-var-requires
const uri = require("../../../screens/imagePage/img_avatar3.png");

describe("List", () => {
  describe("rendering", () => {
    let component: ShallowWrapper;
    beforeEach(() => {
      component = shallow(<List dataSource={
        [
          {
            id: 0,
            text: "Ben",
            source: uri,
          },
        ]
      }
        onPress={() => {
          alert("You tapped the List!");
        }}

      />);
    });
    it("renders", () => {
      expect(component.exists()).toBe(true);
    });
    it("toMatchSnapshot", () => {
      expect(component).toMatchSnapshot();
    });
  });
});
