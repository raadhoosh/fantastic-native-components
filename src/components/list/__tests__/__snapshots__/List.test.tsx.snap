// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List rendering toMatchSnapshot 1`] = `undefined`;
